<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Météo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pBCalculer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInformationGlobal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenue sur l'application météo, veuillez saisir les valeurs demendez</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>332</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="lNbJourMois">
     <property name="text">
      <string>Nombre de jours dans le mois :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lENbJoursMois"/>
    <widget class="QPushButton" name="pBSuivant">
     <property name="text">
      <string>Suivant</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>125</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lInfoTemp">
         <property name="text">
          <string>Température du jours :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lNumJour">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lETemp"/>
     </item>
     <item>
      <widget class="QPushButton" name="pBValider">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="lInfoMoyenne">
       <property name="text">
        <string>Moyenne :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lMoyenne">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lInfoNJF">
         <property name="text">
          <string>Nb Jours froid : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lNbJourFroid">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lInfoJLPF">
           <property name="text">
            <string>Jour le plus froid, n° :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lIdJourLePlusFroid">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lTempFroide">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lInfoDegreeFroid">
           <property name="text">
            <string>degree</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>128</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lInfoNJC">
         <property name="text">
          <string>Nb Jours chaud :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lNbJourChaud">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lInfoJLPC">
           <property name="text">
            <string>Jour le plus chaud, n° :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lIdJourLePlusChaud">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lTempChaude">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lInfoDegreeChaud">
           <property name="text">
            <string>degree</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pBEnregistrer">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInfoEnregister">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>splitter</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pBCalculer</zorder>
   <zorder>lInformationGlobal</zorder>
   <zorder>pBEnregistrer</zorder>
   <zorder>lInfoEnregister</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
